Casks/hem.rb

```diff
@@ -5,11 +5,10 @@
   url "https://dx6pc3giz7k1r.cloudfront.net/hem/versions/#{version}/hem-#{version}.dmg"
   name 'Hem'
   homepage 'https://github.com/inviqa/hem'
-  license :mit
 
   pkg "hem-#{version}.pkg"
 
-  caveats <<-EOS.undent
+  caveats <<-EOS
```

Comment:
so not following the recommended route from the error message of <<~EOS?

Reply:
Updated :)